### PR TITLE
fix: adds support for `releaseCommitMessageFormat`

### LINKED ISSUE #350: releaseCommitMessageFormat Support
This is a bit of a cross-repo (the change will likely occur in [the `conventionalcommits` preset](https://github.com/conventional-changelog/conventional-changelog/tree/master/packages/conventional-changelog-conventionalcommits))  ticket.

`releaseCommitMessageFormat` is not currently supported by the `preset`/`standard-version`. I've linked a PR with a failing test. 

---

~It seems like the fix will be to convert `releaseCommitMessageFormat` the [`message` property](https://github.com/conventional-changelog/standard-version/blob/master/lib/lifecycles/commit.js#L41) in the preset loader, but going that route does seem to couple the preset and standard-version in a not-so-obvious way.~ Added to `standard-version` by aliasing `--message`.

### PR DESCRIPTION
**Pull request by jbottigliero**
- adds support for `releaseCommitMessageFormat` from the spec via `--message` alias
- adds test covering: `.versionrc`, new `--releaseCommitMessageFormat` flag, and current `--message` flag

resolves #350

### CODE REVIEW

**Comment by bcoe** [comment]
@jbottigliero this is looking good to me, perhaps keep this open and we can include the fix?

**Comment by jbottigliero** [comment]
@bcoe – yup! Grabbing this one now.

File: command.js [hunk 1]
```diff
@@ -26,7 +26,7 @@ const yargs = require('yargs')
     default: defaults.infile
   })
   .option('message', {
-    alias: 'm',
+    alias: ['m', 'releaseCommitMessageFormat'],
```
Comment (bcoe) [inline command.js]: maybe add a note that we'll deprecate message in the next `major`, and warn if people use `--message` or `-m`?

**Comment by jbottigliero** [comment]
See #359 for the warning...

**Comment by bcoe** [comment]
@jbottigliero mind rebasing this, this looks great to me.

**Comment by jbottigliero** [comment]
@bcoe – updated, should be good to go.

**Comment by bcoe** [comment]
@jbottigliero thank you!

**Pull request merged**